database/teamD.py

```diff
@@ -45,3 +51,12 @@ def updateTeam(self, team):
         )
 
         self.conn.commit()
+
+    def deletePlayer(self, team):
+    
+        self.cursor.execute(
+            'DELETE  FROM teams WHERE id = ?',
+            ([team.id]) 
```

Comment:
Aquí no necesitas las **[]**.

Reply:
Cuando elimino los corchetes, el Programa lee el id por separado y sale un error que dice... "sqlite3.ProgrammingError: Incorrect number of bindings supplied. The current statement uses 1, and there are 5 supplied.".

Reply:
ok, entonces por ahora dejalo asi